manifests/pipecd/values.yaml

```diff
@@ -373,13 +373,35 @@ prometheus:
 # https://github.com/grafana/helm-charts/tree/main/charts/grafana
 grafana:
   adminPassword: admin
+  dashboardProviders:
+    dashboardproviders.yaml:
+      apiVersion: 1
+      providers:
+        - name: 'Cluster-Stats'
```

Comment:
nit: One vote from me to get rid of the suffix, just use `Cluster`, `Control-Plane`, `Piped`.

Reply:
Alright. I'm gonna adopt it as I feel the same way.